README.md

```diff
@@ -105,6 +105,18 @@ We do that by enabling some special [flags](https://doc.rust-lang.org/cargo/refe
 RUSTFLAGS="-C debug-assertions -C overflow-checks -C debuginfo=2" cargo test --release
 ```
 
+```
+
+## Fuzzing
+
+The `fuzz-smt` fuzz target is designed to test the `Smt` implementation. It generates random SMT entries and updates, and then compares the results of the sequential and parallel implementations.
+
+To run the fuzz target, use the following command:
+
+```shell
+make fuzz-smt
```

Comment:
Might be worth mentioning that one needs to install cargo-fuzz via `cargo install cargo-fuzz`.